git-dependencies

```diff
@@ -62,10 +69,13 @@ def create_parser():
 									 description='Manage dependencies in git repositories',
 									 formatter_class=argparse.RawTextHelpFormatter)
 	parser.add_argument('-r', '--recursive', action='store_true', help='recurse into dependencies')
+	parser.add_argument('-o', '--overrides', help='override dependencies from file')
+	parser.add_argument('-oa', '--override-all', action='store_true', help='enforce complete override (fail if a dependency is not overridden)')
```

Comment:
No, recursive operation is handled by the --recursive switch already.
The --override-all switch is a safety measure, to avoid accidental partial override. Without --override-all, you can override some dependencies and keep others. With --override-all, everything must be overridden or the operation fails.